### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by isaactorres**
This is adding tests to lock in correct functionality

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,12 +34,17 @@
     "watch:build": "onchange 'index.js' --initial -- npm run build -- --continue-on-error",
     "watch": "npm-run-all --parallel --silent watch:*",
     "prepublish": "npm run build",
-    "test": "xo; npm run build"
+    "test": "xo && jest; npm run build"
   },
   "devDependencies": {
+    "@babel/core": "^7.2.2",
```
Comment (isaactorres) [inline package.json:40]: Babel was needed in `devDependencies` so that jest can import the files.

File: package.json [hunk 2]
```diff
@@ -34,12 +34,17 @@
...
+    "@babel/preset-env": "^7.2.3",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (fregante) [inline package.json:42]: There are two babel-core

File: package.json [hunk 3]
```diff
@@ -34,12 +34,17 @@
...
     "bfred-npm-bundler": "^7.1.2",
+    "jest": "^23.6.0",
     "npm-run-all": "^2.3.0",
     "onchange": "^2.5.0",
+    "regenerator-runtime": "^0.13.1",
```
Comment (fregante) [inline package.json]: Is this required?

File: dist/intrinsic-scale.browser.js [hunk 4]
```diff
@@ -1,2 +1,2 @@
 /*! npm.im/intrinsic-scale */
-!function(i){"use strict";function n(i){return function(n,t,c,r){var e,s,u=c/r,a=n/t;return(i?u>a:u<a)?(e=n,s=e/u):(s=t,e=s*u),{width:e,height:s,x:(n-e)/2,y:(t-s)/2}}}var t=n(!0),c=n(!1);i.contain=t,i.cover=c}(this.intrinsicScale=this.intrinsicScale||{});
\ No newline at end of file
+!function(i){"use strict";function n(i){return function(n,t,c,r){var e=c/r,s=n/t,u=n,a=t;return(i?e>s:e<s)?a=u/e:u=a*e,{width:u,height:a,x:(n-u)/2,y:(t-a)/2}}}var t=n(!0),c=n(!1);i.contain=t,i.cover=c}(this.intrinsicScale=this.intrinsicScale||{});
```
Comment (fregante) [inline dist/intrinsic-scale.browser.js]: If you can undo this change it'd be easier to merge

File: .babelrc [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```
Comment (fregante) [inline .babelrc]: Can this be moved to package.json? Example: https://github.com/sindresorhus/refined-github/blob/3737e091b697bf87f0ec4964eb726ab8eaf5ad10/package.json#L86-L103

**Review by fregante — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (isaactorres) [reply package.json:42]: All of these packages are referenced in the [Jest readme](https://jestjs.io/docs/en/getting-started.html#using-babel)

File: package.json (hunk 3 above)
Comment (fregante) [reply package.json]: > Note: Explicitly installing regenerator-runtime is not needed if you use npm 3 or 4 or Yarn

https://jestjs.io/docs/en/getting-started.html#using-babel

File: package.json (hunk 2 above)
Comment (fregante) [reply package.json:42]: That's just sad

Comment (isaactorres) [reply package.json:42]: Maybe it's not necessary if babel < 7 is used. Let me see.

Comment (fregante) [reply package.json:42]: I'd wait for this 

https://github.com/facebook/jest/issues/4842#issuecomment-452017187

**Comment by isaactorres** [comment]
@bfred-it Updated:
- Removed `.babelrc` file
- Removed `regenerator-runtime` package
- Reverted `dist/intrinsic-scale.browser.js` changes

**Comment by fregante** [comment]
Thanks @isaactorres! Appreciated
